test/bigtest/tests/viewContact.e2e.js

```diff
@@ -0,0 +1,40 @@
+import { beforeEach, describe, it } from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import setupApplication from '../helpers/setup-application';
+
+import {
+  ViewContactInteractor,
+} from '../interactors';
+
+describe('View contact', () => {
+  setupApplication();
+
+  const viewContact = new ViewContactInteractor();
+
+  beforeEach(function () {
+    const contact = this.server.create('contact', {
+      categories: ['123'],
```

Comment:
should we move something like this into the factory? so in every place we want contact back-end model - it contains some stub data like this